source/operations/troubleshooting.rst

```diff
@@ -156,6 +158,41 @@ You can scroll vertically through the output for the selected check.
    :alt: MinIO SUBNET's health report for a deployment showing a failed Health Report with details expanded
    :align: center
 
+Logs
+----
+
+Use :mc:`mc admin logs` command to display logs from the command line.
+The command supports type and quantity filters for further limiting logs output.
+
+Optionally, use :ref:`Call Home <minio-troubleshooting-call-home>` to start automatically uploading real time error logs to SUBNET for analysis.
+
+.. _minio-troubleshooting-call-home:
+
+Call Home
+---------
+
+Call Home functionality automates the collection and uploading of diagnostic data or error logs to SUBNET.
+You much have an active SUBNET registration for the deployment to use Call Home.
+
+When enabled, Call Home can send one or both of:
+
+- error logs to SUBNET in real time
+- a new diagnostic report every 24 hours
+
+The diagnostic report upload happens every 24 hours from the time you enable Call Home.
+If needed, you can adjust the frequency of the diagnostic reports and uploads.
+
+Once uploaded, you can view the diagnostic report results or logs through SUBNET as described above, but without the need to manually upload the data yourself.
+Making these records automatically available in SUBNET allows for easy access to manage the health of your deployment.
```

Comment:
Probably helpful to add a blurb that we don't collect PII with callhome and maybe a spare list of example data like disk size, cpu type etc.

Reply:
I agree with Eco's suggestion because that would be the obvious question on the user's mind - what is collected and what is not.

Reply:
Added an admonition block about PII and new sections for the diagnostic and log reports with additional info about what the diagnostic report includes.